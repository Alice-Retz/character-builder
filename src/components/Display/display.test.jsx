import { render } from "@testing-library/react";
import Display from "./Display";

it("should render our display component", () => {
	const { container } = render(
		<Display catchphrases={["tweet", "neigh", "bark"]} />
	);

	expect(container).toMatchSnapshot();
});
