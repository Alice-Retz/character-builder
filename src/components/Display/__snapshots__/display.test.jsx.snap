// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render our display component 1`] = `
<div>
  <div>
    <p>
      tweet
    </p>
    <p>
      neigh
    </p>
    <p>
      bark
    </p>
  </div>
</div>
`;
